### ISSUE TITLE
Migrate the ingestion cronjob to kubernetes job

### ISSUE THREAD

**Issue by pnpavlov** [gardenlinux/glvd#84]
The first prototype for ingesting the data from the public data sources focuses on showcasing how the APIs work and can be used. It is written in Python and planned to run in Linux cronjob directly on the VM. The goal is to package the job in a container and use consistent technologies as the rest of the backend application.

**Comment by pnpavlov** [comment]
@fwilhe , can you please confirm the ingestion job is migrated to kubernetes cron job and can run in container now. If so, please close this issue.

**Comment by fwilhe** [comment]
Yes, the cron job is running on a daily basis in our cluster. The k8s manifest for the cronjob is [here](https://github.com/gardenlinux/glvd/blob/main/deployment/k8s/02_ingestion-job.yaml).

For pure practical reasons I don't recommend running this without restoring a backup from the `gardenlinux/glvd-init` container first, but this is only to avoid long wait times and because there is a chance that an ingestion from a blank state might fail due to limitations of the external apis.
